.circleci/config.yml

```diff
@@ -15,6 +15,17 @@ parameters:
 
 workflows:
   version: 2
+  nightly:
+    triggers:
+      - schedule:
+          # at 7:00 on Monday
```

Comment:
Nit: If it's on Monday's only, should we call it `weekly` instead of `nightly`? Otherwise, it's a bit misleading